app/views/other-resources.html

```diff
@@ -14,6 +14,7 @@
               <ui-select-match placeholder="Choose a resource to list...">{{$select.selected.kind | humanizeKind : true}}</ui-select-match>
               <ui-select-choices repeat="kind in kinds | filter : {kind: $select.search} : matchKind | orderBy : 'kind'">
                 <div ng-bind-html="(kind.kind | humanizeKind : true) | highlight: $select.search"></div>
+                <small ng-bind-html="kind.group | highlight: $select.search" class="text-muted"></small>
```

Comment:
we shouldn't always show this, most of the time it is unnecessary gorp.  I would only show it when it is necessary to distinguish between two Kinds that are the same.  Similar to what we do in the projects dropdown when we see two projects have the same display name.